addins/Cake.Testing.yml

```diff
@@ -0,0 +1,11 @@
+Name: Cake.Testing
```

Comment:
Should be filtered as it is part of Cake and not an addin

Reply:
Done. Cake.Testing added to blacklist